sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTransmitter.cs

```diff
@@ -62,5 +66,10 @@ public async ValueTask<int> TrackAsync(IEnumerable<TelemetryItem> telemetryItems
 
             return itemsAccepted;
         }
+
+        /// <summary>
+        /// This method is provided to inspect the Ingestion Uri in unit tests.
+        /// </summary>
+        internal string GetIngestionUri() => this.applicationInsightsRestClient.GetIngestionUri().ToUri().ToString();
```

Comment:
Is there a way to get rid of it?

Reply:
Not an obvious way :)
The restclient is a private variable inside the Transmitter.
https://github.com/Azure/azure-sdk-for-net/blob/f764848d8db61c658f0a4222294a3df8b33de425/sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTransmitter.cs#L20-L22

Alternatively, I could make this variable internal which would be excellent for the unit tests. But I think that's less desirable because it would be exposed to this full library. I'm open to suggestions here.